src/pages/Survey/index.jsx

```diff
@@ -1,11 +1,23 @@
-import style from './style.module.css'
+import style from './style.module.css';
+import OpenText from '../../components/OpenText';
+import MultipleChoice from '../../components/MultipleChoice';
+
+import data from '../../dataFromAPI/formattedQuestions.json';
 
 function Survey(props) {
     return (
-        <div>
-            <h1>Survey</h1>
+        <div className={style.survey}>
+            {data.questionnaire.map((question, idx) => {
+                if (question.type === 'multiple_choice') {
+                    return <MultipleChoice data={question} key={idx} />;
+                } else if (question.type === 'open_text') {
+                    return <OpenText data={question} key={idx} />;
+                } else {
+                    return null;
+                }
```

Comment:
instead of doing an `if else` everytime, it is enough to do 

```javascript

if (true) {
  return something
}

if ....
```

the code will look cleaner